superduperdb/base/datalayer.py

```diff
@@ -827,13 +817,6 @@ def _build_task_workflow(
                             dep_node,
                             f'{listener.model.identifier}.predict_in_db({listener.uuid})',
                         )
-                    else:
-                        raise Exception(
-                            f'The dependent listener {upstream.uuid}'
-                            ' does not exist.'
-                            'Please make sure to add this listener'
-                            f'`db.add(...)`'
-                        )
```

Comment:
When multiple listeners are chained, there is no need to enforce relationship building in the absence of upstream tasks. Relationships should only be established when upstream and downstream tasks exist concurrently. Therefore, throwing exceptions is unnecessary.